developers/weaviate/configuration/inverted-indexes.md

```diff
@@ -0,0 +1,247 @@
+        language="js"
+    />
+  </TabItem>
+  <TabItem value="js2" label="JS/TS Client v2">
+    <FilteredTextBlock
+      text={TSCodeV2}
+        startMarker="// START FilterIndex"
+        endMarker="// END FilterIndex"
+        language="js"
+    />
+  </TabItem>
+</Tabs>
+
+## indexRangeFilters
+
+The `indexRangeFilters` is a range-based index for filtering by [numerical ranges](/developers/weaviate/release-notes/release_1_26#improved-range-queries). This index is not enabled by default.
+
+Set these indexes on the property level.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCodeV4}
+      startMarker="# START RangeIndex"
+      endMarker="# END RangeIndex"
+      language="py"
+    />
+  </TabItem>
+  <TabItem value="py3" label="Python Client v3">
+    <FilteredTextBlock
+      text={PyCodeV3}
+        startMarker="# START RangeIndex"
+        endMarker="# END RangeIndex"
+        language="py"
+    />
+  </TabItem>
+  <TabItem value="js" label="JS/TS Client v3">
+    <FilteredTextBlock
+      text={TSCodeV3}
+        startMarker="// START RangeIndex"
+        endMarker="// END RangeIndex"
+        language="js"
+    />
+  </TabItem>
+  <TabItem value="js2" label="JS/TS Client v2">
+    <FilteredTextBlock
+      text={TSCodeV2}
+        startMarker="// START RangeIndex"
+        endMarker="// END RangeIndex"
+        language="js"
+    />
+  </TabItem>
+</Tabs>
+
+## bm25
+
+The [`bm25`](/developers/weaviate/config-refs/schema#bm25) ranking algorithm is configured on the collection level. The BM25 algorithm does not specify values for `b` or `k1`. Use these ["free parameters"](https://en.wikipedia.org/wiki/Free_parameter) to tailor the ranking algorithm for your data.
+
+To adjust for document length, modify `b`. Values range from 0 to 1.
+
+To adjust for word frequency within a document, modify `k1`. Values are usually in the range from 0 to 3. There isn't an upper limit.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCodeV4}
+      startMarker="# START BM25Index"
+      endMarker="# END BM25Index"
+      language="py"
+    />
+  </TabItem>
+  <TabItem value="py3" label="Python Client v3">
+    <FilteredTextBlock
+      text={PyCodeV3}
+        startMarker="# START BM25Index"
+        endMarker="# END BM25Index"
+        language="py"
+    />
+  </TabItem>
+  <TabItem value="js" label="JS/TS Client v3">
+    <FilteredTextBlock
+      text={TSCodeV3}
+        startMarker="// START BM25Index"
+        endMarker="// END BM25Index"
+        language="js"
+    />
+  </TabItem>
+  <TabItem value="js2" label="JS/TS Client v2">
+    <FilteredTextBlock
+      text={TSCodeV2}
+        startMarker="// START BM25Index"
+        endMarker="// END BM25Index"
+        language="js"
+    />
+  </TabItem>
+</Tabs>
+
+## Collection level properties
+
+These properties are configured on the collection level.
```

Comment:
I think the text could briefly state what these options do (enable/disable metadata filtering). They could be comments in code or in the prose here. 

The same with the header. I think "Enable metadata filters" or something like that could really help the users find **what** this section does, rather than **where** (collection level) it is configured.